mysql-test/lib/My/SafeProcess/safe_process.cc

```diff
@@ -236,7 +238,8 @@ int main(int argc, char* const argv[] )
   sigaction(SIGCHLD, &sa,NULL);
   sigaction(SIGABRT, &sa_abort,NULL);
 
-  sprintf(safe_process_name, "safe_process[%ld]", (long) own_pid);
+  snprintf(safe_process_name, SAFE_PROCESS_NAME_LEN,
+           "safe_process[%ld]", (long) own_pid);
```

Comment:
Again, don't introduce SAFE_PROCESS_NAME_LEN. It only makes it harder to see that the buffer size is correct for the string being constructed, and that the correct length is passed to sprintf() (as opposed to using sizeof())